### ISSUE TITLE
Bug regarding percent symbol '%' in JS postproc code

### ISSUE THREAD

**Issue by raymond-h** [Hardmath123/nearley#62]
When a `%` symbol is inside of a postprocessor block, it causes an error. It seems the parser doesn't ignore `%` that are found in arbitrary JS code; it consistently complains that there are "no possible parsings" while pointing at whatever symbol - any symbol - after the culprit `%` symbol.

Example:
```js
main -> number {% function(d) { return d[0] + '%' } %}

number -> [\d]:+ {%
	function(d) { return d[0].join(''); }
%}
```

Error:
```
Error: nearley: No possible parsings (@48: ''').
    at Parser.feed (C:\Users\Raymond\AppData\Roaming\npm\node_modules\nearley\lib\nearley.js:219:23)
    at StreamWrapper.write [as _write] (C:\Users\Raymond\AppData\Roaming\npm\node_modules\nearley\lib\stream.js:12:18)
    at doWrite (_stream_writable.js:301:12)
    at writeOrBuffer (_stream_writable.js:288:5)
    at StreamWrapper.Writable.write (_stream_writable.js:217:11)
    at ReadStream.ondata (_stream_readable.js:540:20)
    at ReadStream.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at ReadStream.Readable.push (_stream_readable.js:126:10)
    at onread (fs.js:1679:12)
```

**Comment by JacobEdelman** [comment]
I'm surprised nobody noticed this bug till now. I suppose the best way to fix this would be to ignore ```%``` inside strings and without a ```}``` after them in the JS code.

**Comment by raymond-h** [comment]
That sounds perfect, nothing to add to that.

**Comment by Hardmath123** [comment]
Fixed in latest commit (1.5.0). Sorry this took so long; I was out of town.
